### ISSUE TITLE
clarify correlation between zoom, precision, + human-readable placename in docs

### ISSUE THREAD

**Issue by jywarren** [publiclab/leaflet-blurred-location#98]
We should clarify the exact correlation between zoom and precision and human-readable placename.

See how in the README, we walk through the relationship between precision and decimal places and zoom level; we could do the same (with a table of what precisions relate to what human readable location "scales" like city, state, country) for this. 

We should probably start by making a table which lays out the available human readable place names, so we can propose appropriate precision values for each. 

So something like:

Zoom level | Lat/lon coordinate precision | Human-readable placename
--|----|---
0 | `x0.0` | planet
5 | `x0.0` | state, province, country
6 | `x.0` | state, province, region
10 | `0.x` | city, postal code
13 | `0.0x` | neighborhood
16 | `0.00x` | block

```
Zoom level | Lat/lon coordinate precision | Human-readable placename
--|----|---
0 | `x0.0` | planet
5 | `x0.0` | state, province, country
6 | `x.0` | state, province, region
10 | `0.x` | city, postal code
13 | `0.0x` | neighborhood
16 | `0.00x` | block
```

**Comment by jywarren** [comment]
@ebarry does this make sense to you?

**Comment by ebarry** [comment]
Yes, this makes sense, and you're talking about for the wiki, right?

**Comment by jywarren** [comment]
Both the wiki and the README file could feature this :-)

wiki maybe at https://publiclab.org/wiki/location-privacy

**Comment by jywarren** [comment]
Related to #134

**Comment by mridulnagpal** [comment]
@jywarren Should we start with this as well?

**Comment by jywarren** [comment]
That would be GREAT -- and also adding a link from the location input pop-up to this page so that people can easily find it!

**Comment by mridulnagpal** [comment]
Where do you think we should put that link? In a button?
